### DISCUSSION TITLE
Trusted host configuration

### DISCUSSION THREAD

**Discussion by HuM4NoiD** [pdm-project/pdm#1654] [Q&A] [answered]
Does pdm pick up trusted hosts from `pip.conf` or is there an explicit property to be set in pdm config?
I tried to add a simple http index on a local network with:
```
❯ pdm config pypi.url "http://local-ip:port/repository/pypi-group/simple"
Skipping http://local-ip:port/repository/pypi-group/simple/pdm/ for not being trusted, please add it to `trusted_hosts` list
```
`local-ip` is a valid ip address

Tried setting `trusted_host` property:
```
❯ pdm config pypi.trusted_hosts "local-ip"
[PdmUsageError]: Set index config with pypi.{name}.{attr}
```
and
```
❯ pdm config trusted_hosts "local-ip"
[NoConfigError]: 'No such config item: trusted_hosts'
```

pip has `global.trusted-hosts` set with the appropriate values

**Comment by frostming** [comment, marked as answer]
You are looking for `verify_ssl` option: https://pdm.fming.dev/latest/usage/project/#configure-pypi-indexes



> this appends `/pdm/` in the end. Is this intentional

Yes, the package name is appended because you are requesting for it. The dependencies may contain `pdm`.

**Reply by boholder** [reply]
New doc url: https://pdm-project.org/latest/usage/config/#configure-the-package-indexes
